cmake/modules/hunter_windows_to_snf_filename.cmake

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2015 Sumedh Ghaisas, Ruslan Baratov
+# All rights reserved.
+
+# CMAKE function to convert windows log filename to 8.3 filename
+# For reference check http://stackoverflow.com/a/10227715/2288008
+function(hunter_windows_to_snf_filename path new_path_variable)
+  set(conversion_cmd
+    "cmd" 
+    "/c" 
+    "for" 
+    "%A" 
+    "in" 
+    ("${path}") 
+    "do" 
+    "@echo"
+    "%~sA"
+	)
+  
+  execute_process(
+    COMMAND ${conversion_cmd}
+    RESULT_VARIABLE conversion_result
+    OUTPUT_VARIABLE converted_path
+  )
+  
+  if(NOT conversion_result EQUAL 0)
+    hunter_fatal_error("Command to convert path ${path} to 8.3 Filename failed. Command is ${conversion_cmd}")
+  endif()
+  
+  string(REGEX REPLACE "\n" "" converted_path ${converted_path})
```

Comment:
You can use [OUTPUT_STRIP_TRAILING_WHITESPACE](https://cmake.org/cmake/help/v3.0/command/execute_process.html) to avoid this step.